doc/cabal-project.rst

```diff
@@ -194,8 +194,8 @@ Specifying Packages from Remote Version Control Locations
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Starting with Cabal 2.4, there is now a stanza
-``source-repository-package`` for specifying packages from an external
-version control.
+``source-repository-package`` for specifying remote packages that cabal will vendor from an external version control system.
+This allows sharing of packages across different projects.
```

Comment:
Although it's not added by you, I'd like this to be clarified because I don't understand this comment. Sharing also works without the stanza with normal Hackage packages: if two different projects depend on the same package that gets resolved to the same version, they will use the single instance of the package residing in the store.

Reply:
This line doesn't make much sense to me either. I'm using this, so i can (temporarilly!) use a modified version of upstream packages, while i wait on them to accept PRs. I'm good with dropping this line, or substituting "this allows use of a remote package in version control, rather than looking for a package in Hackage."

Reply:
Yes, that sounds good.

Reply:
```suggestion
This allows use of a remote package in version control, rather than looking for a package in Hackage.
```